app/views/propertyLinking/upload_result.scala.html

```diff
@@ -0,0 +1,160 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import actions.propertylinking.requests.LinkingSessionRequest
+@import binders.propertylinks.EvidenceChoices.EvidenceChoices
+@import binders.propertylinks.EvidenceChoices
+@import config.ApplicationConfig
+@import models.{EvidenceType, _}
+@import models.attachment.Attachment
+@import models.upscan.FileStatus.FileStatus
+@import models.upscan._
+
+
+@this(govukTag: GovukTag, govukSummaryList: GovukSummaryList, mainLayout: views.html.mainLayout, formWithCSRF: FormWithCSRF, govukButton: GovukButton, govukErrorSummary: GovukErrorSummary)
+
+@(
+        evidenceType: EvidenceType,
+        evidenceChoice: EvidenceChoices,
+        submissionId: String,
+        uploadedFiles: Map[String, UploadedFileDetails] = Map(),
+        status: FileStatus,
+        fileReference: Option[String],
+        optAttachment: Option[Attachment])(implicit request: LinkingSessionRequest[_], messages: Messages, config: ApplicationConfig)
+
+@attachmentFileName = @{
+    optAttachment.map(_.fileName).getOrElse("")
+}
+
+@attachmentDownloadLink = @{
+    optAttachment.flatMap(_.scanResult.flatMap(_.downloadUrl))
+}
+
+@attachmentFileNameHtml = {
+@if(attachmentDownloadLink.isDefined) {
+    <a class="govuk-link" href="@attachmentDownloadLink.get">@attachmentFileName</a>
+} else {
+    <a class="govuk-link" href=""></a>
+}
+}
+
+@title = @{
+    (request.ses.clientDetails, evidenceType) match {
+        case (Some(_), Lease) => {
+            messages("uploadEvidence.lease.client.title")
+        }
+        case (Some(_), License) => {
+            messages("uploadEvidence.license.client.title")
+        }
+        case (Some(_), RatesBillType) => {
+            messages("uploadRatesBill.show.client.title")
+        }
+        case (Some(_), _) => {
+            messages(s"uploadEvidence.$evidenceType.title")
+        }
+        case (None, Lease) => {
+            messages("uploadEvidence.lease.title")
+        }
+        case (None, License) => {
+            messages("uploadEvidence.license.title")
+        }
+        case (None, RatesBillType) => {
+            messages("uploadRatesBill.show.title")
+        }
+        case (None, _) => {
+            messages(s"uploadEvidence.$evidenceType.title")
+        }
+    }
+}
+
+@tag(status: FileStatus) = @{
+    status match {
+        case _@FileStatus.UPLOADING => govukTag(Tag(content = Text(messages("fileUploadResult.uploading")), classes = "govuk-tag--yellow", attributes = Map("id" -> "uploadStatus")))
+        case _@FileStatus.READY => govukTag(Tag(content = Text(messages("fileUploadResult.uploaded")), classes = "govuk-tag--green", attributes = Map("id" -> "uploadStatus")))
```

Comment:
is it right that the READY status maps to fileUploadResult.uploaded?

Reply:
Yep, one of the issues with the design of the ticket is that we display uploading/uploaded which aren't accurate to whats happening but yes in this case Ready = Uploaded